enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -20,6 +20,34 @@ or using yarn:
 yarn add @metabase/embedding-sdk-react
 ```
 
+## Which SDK version to use
+
+The SDK version tracks with the Metabase version.
+
+So, for example, if you're on Metabase version `1.51.x`, you should use the SDK with the tag `51-stable`.
+
+See [Versioning](https://www.metabase.com/docs/latest/embedding/sdk/version).
+
+## Installing Metabase Enterprise Edition
+
+You can run Metabase Pro on a Cloud plan (with a free trial)](https://www.metabase.com/pricing).
+
+Or run it locally. Here's a docker one-liner:
+
+```sh
+docker run -d -p 3000:3000 --name metabase metabase/metabase-enterprise:latest
```

Comment:
Since this will grab the latest version of Metabas and this changes from time to time, I wonder if we should flip the order to first have a MB and then install the npm package based on its version.